main.py

```diff
@@ -51,7 +55,7 @@ def extract_transform_and_load_covid_data(post_to_google_sheets=True):
             debugging of data processing
 
     """
-    print("Starting to ETL...")
+    logger.info("Starting to ETL...")
```

Comment:
hell yea! lol. 

We have a bunch of other `prints` we could remove as well in this PR or future work...

Reply:
yep for sure. I'll see if it's an easy copy -replace